go-app-chw.js

```diff
@@ -485,6 +519,28 @@ go.utils = {
         });
     },
 
+    build_nurse_jembi_json: function(im, contact, user, type) {
+        var JSON_template = {
+            "mha": 1,
+            "swt": go.utils.get_swt(im),
+            "dmsisdn": user.msisdn,
+            "cmsisdn": contact.msisdn,
+            "type": go.utils.get_subscription_type(type),
+            "encdate": go.utils.get_timestamp(),
+            "faccode": contact.extra.nc_faccode || null,
+            "dob": go.utils.get_dob(contact) || null,
+            "id": go.utils.get_nurse_id(contact),
+            "optoutreason": go.utils.get_nurse_optoutreason(contact)
+        };
+        return JSON_template;
+    },
+
+    jembi_nurse_optout_send_json: function(contact, user, type, im, metric_prefix) {
+        var built_json = go.utils.build_nurse_jembi_json(im, contact, user, type);
+        return go.utils
+            .jembi_json_api_call('post', null, built_json, 'optout', im);
```

Comment:
Double-check this on response from Pierre that optout endpoint not diff for nurses